sdk/apis/kubermatic/v1/settings.go

```diff
@@ -43,6 +43,13 @@ type KubermaticSetting struct {
 // allowedOperatingSystems defines a map of operating systems that can be used for the machines.
 type allowedOperatingSystems map[providerconfig.OperatingSystem]bool
 
+type ClusterBackupOptions struct {
+	// DefaultChecksumAlgorithm allows setting a default checksum algorithm used by Velero for uploading objects to S3.
+	//
+	// Optional
+	DefaultChecksumAlgorithm *string `json:"defaultChecksumAlgorithm,omitempty"`
```

Comment:
Unless you want to distinguish between nil and "", a pointer is not necessary here.